src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+                    //get the task description
+                    int slashIndex = words.indexOf('/');
+                    tasks[countTask] = new Event(words.substring(6, slashIndex-1), words.substring(slashIndex + 3));
+                    addTask(tasks[countTask], countTask);
+                    countTask++;
+                }
+            }
+
+            //remove the task you don't want to keep
+            else if (words.startsWith("remove")) {
+                if (words.length() <= 7) {
+                    System.out.println("     Sorry, I don't know which task you want to remove.");
+                    printLine();
+                } else {
+                    String task = words.substring(7);
+                    int totalNum = countTask;
+                    for (int i = 0; i < countTask; i++) {
+                        String t = tasks[i].description;
+                        if(t.equals(task)) {
+                            System.out.println("     I've removed this task:");
+                            System.out.println("       " + tasks[i]);
+                            tasks[i] = tasks[countTask-1];
+                            tasks[countTask-1] = null;
+                            countTask--;
+                            System.out.println("     Now you have " +  countTask + " tasks in the list.");
+                            printLine();
+                            break;
+                        }
+                    }
+                    if (totalNum == countTask) {
+                        System.out.println("     I can't find this task in your list.");
+                        printLine();
+                    }
+                }
+            }
+
+            //mark the task as done
+            else if (words.startsWith("done") && words.length() == 6) {
+                String taskIndex = words.substring(5, 6);
+                int index = Integer.parseInt(taskIndex);
+
+                //mark as done
+                finishTask(tasks[index - 1]);
+            }
+
+            //other undefined task name
+            else {
+                System.out.println("     Sorry,this task name is not allowed");
+                printLine();
+            }
+        }
+    }//end main
+
+    public static void greetWords() {
+        printLine();
+        System.out.println("     Hello! I'm Duke");
+        System.out.println("     What can I do for you?");
+        printLine();
+    }
+
+    public static void forgetName() {
+        System.out.println("     Oh no, you forget to add the name for your task.");
+        printLine();
+    }
+
+    public static void printLine() {
+        System.out.print("     -");
+        for(int i=0; i<60; i++) System.out.print('-');
+        System.out.println("-");
+    }
+
+    public static void listTasks(Task[] tasks,int countTask) {
+        if(countTask == 0) {
+            System.out.println("     You have not added any task into your list.");
+        }
+        else {
```

Comment:
I think if-else standards need to be in this form: 
if (...) {
... 
} else if(...) {
...
}

Reply:
this part I also forgot, thank you!